R/derive_var_atoxgr.R

```diff
@@ -214,6 +221,11 @@ derive_var_atoxgr_dir <- function(dataset,
       # check variables required in criteria exist on data
       assert_data_frame(grade_this_filter, required_vars = exprs(!!!syms(list_of_vars)))
 
+      if ("BNRIND" %in% list_of_vars) {
```

Comment:
if the list of variables is fixed (`Alanine aminotransferase increased`, `Alkaline phosphatase increased`, `Aspartate aminotransferase increased`, `Blood bilirubin increased` and `GGT Increased`) should we do anything to check their presence, alert user to missing or some other skullduggery?

Reply:
If BNRIND is a required variable, as it is for those 5 lab tests, then as well as checking for existence on line 222, I then check that `abnormal_indicator` is not NULL.